src/app/components/mno-address/mno-address-input/mno-address-input.html

```diff
@@ -31,4 +31,10 @@
       </select>
     </div>
   </div>
+  <div class="form-group">
+    <div class="col-sm-6 address-field">
+      <label class="address-sub-label">{{ 'mnoe_admin_panel.components.mno-address-input.phone' | translate }}</label>
+      <input type="text" class="form-control" ng-model="address.phone"/>
```

Comment:
Phone will just be a text input right? I think this could possibly cause some confusion for the user when it comes to country code. Should we have a phone_country_code attr on the address model in mnohub? I'm thinking we should have it set-up just like the User model in MnoHub.